commands.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"context"
+	"os/exec"
+
+	"github.com/spf13/viper"
+)
+
+// Creates a command that can be used to run docker-compose.
+func DockerComposeCommand(cfg *viper.Viper, args ...string) *exec.Cmd {
```

Comment:
This is nitpicky so obviously feel free to ignore, but this is 99% duplicated with `DockerComposeCommandContext`, and `DockerCommand` is only used by this function. This function itself is only used two places. I suppose it'd only save a little duplication, but maybe it'd be better to just change those two calls to `DockerComposeCommandContext(cfg, ctx.Background(), ...)` and delete these two functions.

Reply:
That seems reasonable; I'll update that. Thanks!

Reply:
I went with a slightly lazier approach. I left `DockerComposeCommand` and `DockerCommand` in place but modified them to call their counterparts with `context.Background()` as the context.